data-prepper-plugins/parse-json-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/parsejson/ParseJsonProcessorConfig.java

```diff
@@ -31,6 +31,9 @@ public class ParseJsonProcessorConfig {
     @JsonProperty("tags_on_failure")
     private List<String> tagsOnFailure;
 
+    @JsonProperty("overwrite_if_key_exists")
```

Comment:
Should this be `overwrite_if_destination_exists` to align with the naming in this processor?

Reply:
I'm open to the change. But one thing to consider is that destination can be unset, in that case, key-value pairs are being written to event root, and this added option controls whether those key-value pairs overwrite existing keys at root.

Reply:
Will update. This makes sense.

Reply:
@oeyh , You need to change the value in line 34 as well.

Reply:
Done.